appengine-java8/helloworld-springboot/README.md

```diff
@@ -84,7 +84,7 @@ These settings should be revisited for production use.
 
 ## Deploy to App Engine flexible environment
 
-1. `mvn appengine:deploy`
+1. `mvn clean package appengine:deploy -Dapp.deploy.projectId=projectId`
```

Comment:
Is app.deploy.projectId really required?  I thought you would just use GCLOUD_CONFIG

Reply:
Project Id is required. I'm having the use pass it in as an arg instead of changing the pom file.